appinventor/appengine/src/com/google/appinventor/client/wizards/youngandroid/NewYoungAndroidProjectWizard.java

```diff
@@ -43,7 +47,32 @@ public NewYoungAndroidProjectWizard() {
     // Initialize the UI
     setStylePrimaryName("ode-DialogBox");
 
-    projectNameTextBox = new LabeledTextBox(MESSAGES.projectNameLabel());
+    projectNameTextBox = new LabeledTextBox(MESSAGES.projectNameLabel(), new Validator() {
+      @Override
+      public boolean validate(String value) {
+        if(!value.matches("[A-Za-z][A-Za-z0-9_]*") && value.length() > 0) {
```

Comment:
@halatmit What's the best way to test if it's working correctly with international characters? I added a regex to account for this today, but I want to make sure I'm properly testing it.